src/features/search/pages/modals/VenueModal/useVenueModal.native.test.tsx

```diff
@@ -74,16 +102,19 @@ describe('useVenueModal', () => {
       result.current.doSetSelectedVenue(venue)
     })
 
-    expect(result.current.venueQuery).toHaveLength(venue.label.length)
-
     await act(async () => {
       result.current.doResetVenue()
     })
 
-    expect(result.current.isQueryProvided).toBeFalsy()
-    expect(result.current.shouldShowSuggestedVenues).toBeFalsy()
-    expect(result.current.isVenueSelected).toBeFalsy()
-    expect(result.current.venueQuery).toHaveLength(0)
+    const {isQueryProvided,
+      shouldShowSuggestedVenues,
+      isVenueSelected,
+      venueQuery} = result.current
+
+    expect({isQueryProvided,
+      shouldShowSuggestedVenues,
+      isVenueSelected,
+      venueQuery}).toStrictEqual(falsyState) 
```

Comment:
je propose, je ne sais pas si ça sera beaucoup mieux, d'extraire une fonction


```suggestion
    expectToBeFalsyVenue(result.current) 
```

le nom de la fonction est pas top : trop technique

je ne sais pas ce que ça fait fonctionnellement, donc je n'ai pas mieux à proposer